packages/shared/src/hooks/chat/useChatStream.ts

```diff
@@ -62,10 +62,11 @@ export const useChatStream = (): UseChatStream => {
         );
       };
 
-      const trackErrorEvent = () => {
+      const trackErrorEvent = (code: SearchChunkErrorCode) => {
         trackEvent({
           event_name: AnalyticsEvent.ErrorSearch,
           target_id: streamId,
+          extra: JSON.stringify({ code }),
```

Comment:
@idoshamun this adds an extra.code value on the error analytics event for all kinds of error events in the Search.

Reply:
Perfect!